extensions/src/Smithy.Identity.Abstractions/Smithy.Identity.Abstractions.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <AssemblyName>Smithy.Identity.Abstractions</AssemblyName>
+    <GenerateAssemblyTitleAttribute>false</GenerateAssemblyTitleAttribute>
+    <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
+    <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
+    <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
+    <GenerateAssemblyDescriptionAttribute>false</GenerateAssemblyDescriptionAttribute>
+    <GenerateAssemblyCopyrightAttribute>false</GenerateAssemblyCopyrightAttribute>
+    <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
+    <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
+    <SignAssembly>True</SignAssembly>
+  </PropertyGroup>
+
+  <Choose>
+    <When Condition=" '$(AWSKeyFile)' == '' ">
+      <PropertyGroup>
+        <AssemblyOriginatorKeyFile>..\..\..\sdk\awssdk.dll.snk</AssemblyOriginatorKeyFile>
```

Comment:
Because we might extract this into a separate repo and product sometime in the future I wonder if we should have a separate `snk` file for Smithy extensions. If we think X years in future and .NET Smithy being own standalone product it would be weird to have it share the AWS SDK `snk` file.

Reply:
That makes sense, I created a new `snk` file in https://github.com/aws/aws-sdk-net/pull/3330/commits/8d996a29a697a419323c524209c5af250b4d5280 (using the `sn` command described in https://learn.microsoft.com/en-us/dotnet/standard/assembly/create-public-private-key-pair).